functions/menu.bash

```diff
@@ -25,12 +25,14 @@ Note backups are NOT active per default so remember to set them up right at the
 }
 
 show_main_menu() {
-  choice=$(whiptail --title "Welcome to the openHABian Configuration Tool $(get_git_revision)" --menu "Setup Options" 18 116 11 --cancel-button Exit --ok-button Execute \
+  choice=$(whiptail --title "Welcome to the openHABian Configuration Tool $(get_git_revision)" --menu "Setup Options" 20 116 13 --cancel-button Exit --ok-button Execute \
   "00 | About openHABian"        "Information about the openHABian project and this tool" \
   "" "" \
   "01 | Select Branch"           "Select the openHABian config tool version (\"branch\") to run" \
   "02 | Upgrade System"          "Upgrade all installed software packages (incl. openHAB) to their latest version" \
   "03 | openHAB Stable"          "Install or upgrade to the latest stable release of openHAB 2" \
+  "04 | Upgrade to openHAB 3"    "Upgrade to openHAB 3 testing milestone (BETA !)" \
```

Comment:
Should we force switch to OH3 Branch when this option is selected?

Reply:
I was wondering, too.
I migrate when the user selects to switch to openHAB3 branch but don't when he switches from there elsewhere (back down), that's how the upgrade was meant to be triggered. But we had no downgrade path (as we cannot be sure if a specific branch is OH 2 or 3) so I introduced options 04 and 05
On 2nd thought yes we must switch branch or else future actions triggered from the tool will fail.